### ISSUE TITLE
dataset.py

### ISSUE THREAD

**Issue by zneha** [rdcolema/tensorflow-image-classification#18]
On a very general note, could you just in words say what is happening in this python script dataset.py?

Also, when I download the dataset from kaggle, I do not see dogs and cats in two separate folders in the training data folder (referring back to dataset.py which takes in input this way). Do we make separate folders under training folder for cats and dogs?

**Comment by rdcolema** [comment]
The dataset.py scripts are used for loading, batching, and normalizing the training images. The images are divided into a training set, a validation set, and a test set. 

Regarding the Kaggle data, the splits into separate folders have to be done manually, but this should be relatively straightforward with either a bash or python script.
